### PR TITLE
10 sort articles

### PR DESCRIPTION
**Pull request by HarCass**

### CODE REVIEW

File: __tests__/utils.test.js [hunk 1]
```diff
@@ -12,4 +12,23 @@ describe('formatDate()', () => {
         result = formatDate('2020-10-18T01:26:00.000Z');
         expect(result).toBe('18/10/2020 01:26');
     });
+});
+
+describe('formatStrToUpper()', () => {
```
Comment (callumu21) [inline __tests__/utils.test.js:17]: Nice unit testing of a util function

File: src/utils/index.js [hunk 2]
```diff
@@ -2,4 +2,17 @@ exports.formatDate = (dateStr) => {
     if(!dateStr.length) return '';
     const formattedDate = `${dateStr.slice(8, 10)}/${dateStr.slice(5, 7)}/${dateStr.slice(0, 4)} ${dateStr.slice(11, 16)}`
    return formattedDate;
+}
+
+exports.formatStrToTitle = (str) => {
```
Comment (callumu21) [inline src/utils/index.js:7]: Is there a reason why you have a copy of this function in index.js, as well as in its own file? The same for formatDate. If not, then you should only need one version of it.

**Review by callumu21 — APPROVED** [review]
Nice implementation, Harry! Just a few notes for refactoring:

1. I would have your dropdowns for order_by and sort_by set to a default value on initial render. These values are being used without a user selecting them on initial render as the default query is to return all articles in reverse chronological order, so best to show the user that's what's happening.
2. I'd also look at changing the default option you have for your select fields and how you're displaying it. If I select Desc as an order query, for example, it shows up as the first option, but the Desc option is also still visible. This shouldn't be the expected behaviour.
3. Double check at your util functions. It seems as though you've repeated them in /src/utils/index.js and in individual files as well. If there isn't a reason for this, then best to only have the one source of truth.

Once you've made these changes, this should be good to merge without another PR resubmit 👍

File: src/utils/index.js (hunk 2 above)
Comment (HarCass) [reply src/utils/index.js:7]: Jest would not allow ESModule imports and the components were not allowing require imports. I could not figure out a way to get it working both ways, so decided to make two copies of the utils.

**Pull request merged**